index.js

```diff
@@ -16,14 +16,15 @@ const isCI =
 let enabled =
   !isDisabled && (isForced || isWindows || isCompatibleTerminal || isCI)
 
-const raw = (open, close, searchRegex, replaceValue) => (s) =>
-  enabled
-    ? open +
-      (~(s += "").indexOf(close, 4) // skip opening \x1b[
-        ? s.replace(searchRegex, replaceValue)
-        : s) +
-      close
-    : s
+const raw = (open, close, searchRegex, replaceValue) => (s) => {
+  if (!enabled) return s
+  if (!(s += "")) return ""
+  return open +
```

Comment:
Could we avoid the `return` statement?

Reply:
Done; I've replaced it with a ternary expression